### PR TITLE
Added support for multipart/form-data

### PR DESCRIPTION
**Pull request by dejvidecz**
If request is multipart/form-data - code will generates solution with FormData.
image

I have also tried to update angular service generator, but i have no angular project to verify this solution. According to chatgpt and my "common sense" - i believe it will work (Just added formData).

Tests are updated and added new test.

Please check it and let me know if it is ok for you :)

### CODE REVIEW

File: apps/cli/src/tests/__snapshots__/generate_petstore_cookies.ts [hunk 1]
```diff
@@ -424,7 +429,10 @@ export const postPetPetIdUploadImagePath = (petId: number) => `/pet/${petId}/upl
 
 export const postPetPetIdUploadImage = (petId: number, headers = new Headers()): 
 	Promise<PostPetPetIdUploadImageFetchResponse> => {
-	return apiPost(`${getApiUrl()}${postPetPetIdUploadImagePath(petId)}`, {}, headers) as Promise<PostPetPetIdUploadImageFetchResponse>;
+	
+    //multipart/form-data  
+    const formData = new FormData();
```
[the remark this answers is not in the document]
Comment (dejvidecz) [reply apps/cli/src/tests/__snapshots__/generate_petstore_cookies.ts:434]: > This is empty intentionaly or something is not implemented, yet?

I aimed on OpenApi 3 with this syntax:
image

Thats why this example has not changed.
Should i add support?

[the remark this answers is not in the document]
Comment (gaboe) [reply apps/cli/src/tests/__snapshots__/generate_petstore_cookies.ts]: No need, thanks

**Comment by gaboe** [comment]
image


nswag test is not generating code, that can be compiled. Can you please fix this? I should i compile typescript test as part of that test suite :D

**Comment by gaboe** [comment]
i have added tsc script for this at root level

**Pull request merged**